agent/lib/kontena/load_balancers/configurer.rb

```diff
@@ -76,6 +76,8 @@ def ensure_config(container)
 
     # @param [String] service_name
     def remove_config(service_name)
+      return if service_name.to_s.empty?
```

Comment:
Probably yes, this is now called via notifications.. raising error would only crash this actor.